src/main/java/ca/warp7/frc2020/subsystems/ControlPanelSpinner.java

```diff
@@ -9,21 +9,99 @@
 
 import ca.warp7.frc2020.lib.motor.MotorControlHelper;
 import com.revrobotics.CANSparkMax;
+/*
+    Colour sensor documentation link
+    http://www.revrobotics.com/content/sw/color-sensor-v3/sdk/docs/javadoc/com/revrobotics/ColorSensorV3.html
+*/
 import com.revrobotics.ColorSensorV3;
 
 import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.util.Color;
 import edu.wpi.first.wpilibj2.command.Subsystem;
 
 import static ca.warp7.frc2020.Constants.kControlPanelManipulatorID;
 
 public final class ControlPanelSpinner implements Subsystem {
+    
+    public ControlPanelSpinner(){
+        testRgbToHue();
+    }
+
+    private int count = 0;
+
     private static ControlPanelSpinner instance;
 
     public static ControlPanelSpinner getInstance() {
         if (instance == null) instance = new ControlPanelSpinner();
         return instance;
     }
 
+    @Override
+    public void periodic() {
+        if(count % 5 == 0){
+            //System.out.println("Going...going..going...");
+        }
+        count++;
+    }
+
+    /*
+        Converts a 0-1 R G and B value to it's HSL hue value.
+        Depending on what value is the highest, we use a difrent formula.
+    */
+    public static int rgbToHue(final double R, final double G, final double B){
```

Comment:
This looks like a pure function instead of a function that depends on state - it would probably be better to not have it in this file. Instead, put it in a file under the `ca.warp7.frc2020.lib` package and the file could be called something like `ColorUtil`.

Again I would ask how will this function be used?